src/index.ts

```diff
@@ -241,6 +247,23 @@ const vimPlugin = ViewPlugin.fromClass(
   },
   {
     eventHandlers: {
+      copy: function(e: ClipboardEvent, view: EditorView) {
+        if (!this.waitForCopy) return;
+        this.waitForCopy = false;
+        Promise.resolve().then(() => {
```

Comment:
why is this wrapped in a `Promise.resolve()`?

Reply:
Codemirror doesn't have a way to handle copy event after the default handler, so we need some short timeout to clear selection after codemirror handler finishes updating clipboardData.